SR2020/2020-05-22-virtual-competition-transition.md

```diff
@@ -0,0 +1,26 @@
+---
+to: sr2020-teams
+subject: Transitioning to a virtual competition
+---
+
+Hi all,
+
+In light of recent events, and the uncertainty it has brought, we're unfortunately cancelling the in-person competition for Student Robotics 2020.
+
+## Virtual Competition
+
+The virtual competition event will take place over a few weekends in July, with a few weekends of league matches, and a knockout final.
```

Comment:
Perhaps "sessions" rather than "weekends"?
At the moment this reads like the whole weekend might be filed with matches, but I don't think that's what it's trying to say.